get-tom-observations/main.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sqlalchemy
+from flask import jsonify
+
+# Set the following variables depending on your specific
+# connection name and root password from the earlier steps:
+connection_name = os.getenv('CONNECTION_NAME')
+db_password = os.getenv('DB_PASSWORD')
+db_name = os.getenv('DB_NAME')
+
+db_user = os.getenv('DB_USER')
+driver_name = 'postgres+pg8000'
+query_string = dict({"unix_sock": "/cloudsql/{}/.s.PGSQL.5432".format(connection_name)})
```

Comment:
You should use either `dict` or `{}` for creating a dictionary. Here you are passing `{}` to `dict()`, which is redundant.

```suggestion
query_string = dict(unix_soc=f"/cloudsql/{connection_name}/.s.PGSQL.5432")
```